tests/test_dtype.py

```diff
@@ -292,3 +292,35 @@ def kernel(A, B, C, O):
     f(hcl_A, hcl_B, hcl_C, hcl_O)
 
     assert np.array_equal(hcl_O.asnumpy(), np_G)
+
+# def test_dtype_struct_kernel():
```

Comment:
What's this?

Reply:
This test case is for data struct used in HeteroCL module. It does not work so I commented it out.